README.md

```diff
@@ -1,4 +1,152 @@
-# Marsha
+# Marsha, a FUN LTI video provider 🐠
 
-> 🐠 A FUN LTI video provider
+[![CircleCI](https://circleci.com/gh/openfun/marsha/tree/master.svg?style=svg)](https://circleci.com/gh/openfun/marsha/tree/master)
 
+## Overview
+
+`Marsha` is a video management & playback service. It is intended to be operated independently: it's like having your very own YouTube for education.
+
+Instructors & organizations can use Marsha to upload and manage their videos (and associated files, such as subtitles) directly from a course as they are creating it.
+
+Once the course is published, learners simply see a video player in the course.
+
+This seamless integration works with any LMS (`Open edX`, `Moodle`, ...) thanks to the LTI standard for interoperability.
+
+Here is what `Marsha` offers out of the box:
+
+- automatic transcoding of videos to all suitable formats from a single instructor-supplied file;
```

Comment:
I was trying to understand what configuration file the instructor was providing for formats.
Maybe "starting from a video file uploaded by the instructor" ?